pages/journal.js

```diff
@@ -3,16 +3,320 @@ import Helmet from 'react-helmet';
 import { config } from 'config';
 
 class Journal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeTab: 0,
+    }
+  }
+  
   render () {
+	const tabData = [
```

Comment:
Take the static data and move it to it's own config file for the page, then import them in place of this